server/src/main/java/keywhiz/JooqHealthCheck.java

```diff
@@ -35,7 +35,7 @@
 
   private static final Logger logger = LoggerFactory.getLogger(JooqHealthCheck.class);
 
-  enum OnFailure {
+  public enum OnFailure {
     RETURN_UNHEALTHY, LOG_ONLY
   }
```

Comment:
This is referenced by a Module in a different package. Although, Keywhiz has a pattern of locating modules in the same folder as their dependencies (e.g. CryptoModule), so perhaps it makes sense to move the module out of keywhiz.inject and into keywhiz.